apps/wallet-mobile/src/components/Clipboard/ClipboardProvider.tsx

```diff
@@ -24,12 +24,12 @@ export const ClipboardProvider = ({children}: Props) => {
   const styles = useStyles()
   const [copied, setCopied] = React.useState<CopiedState | null>(null)
   const {height, width} = useWindowDimensions()
-  const copy: ClipboardContext['copy'] = ({text, feedback = 'Copied', event}) => {
+  const copy: ClipboardContext['copy'] = ({text, feedback = 'Copied', event, offsetX = 0, offsetY = 0}) => {
     Clipboard.setString(text)
     setCopied({
       feedback,
-      locationY: event ? event.nativeEvent.pageY - 50 : height * 0.85,
-      locationX: (event?.nativeEvent.pageX ?? width * 0.5) - feedback.length * 4,
+      locationY: (event ? event.nativeEvent.pageY - 50 : height * 0.85) + offsetY,
```

Comment:
Seems like a lot of extra props just to not clip it. We could calculate here the screen size as put it as a max, so it doesn't overflow it.
Alternatively, if you don't pass the `event`, the message shows in the middle bottom part of the screen instead of on top of the button.

Reply:
To make the exact calculations you need the feedback text size before rendering. Looking for a workaround

Reply:
I've tried to get the approximate size of the feedback text in several different ways but it's highly unreliable.

Let me know if you can think of a better solution, if not, I don't think this solution is that problematic TBH

Reply:
@banklesss try now

Reply:
yeah it works thanks